content/09.evolution.md

```diff
@@ -69,31 +69,98 @@ Research in response to prior outbreaks of HCoV-borne infections, such as SARS a
+ABO blood type has been found to modulate susceptibility to other pathogens [@doi:10.1001/jama.2020.16516].
+While ABO blood type is a genetic trait, it is more easily evaluated than the genetic regions discussed above because of the simple relationship between genetic variants and phenotype.
+The possibility for an association between blood type and COVID-19 infection was raised early in the pandemic in a preprint that reported associations in 2,173 patients in Wuhan and Shenzhen, China [@doi:10.1101/2020.03.11.20031096].
+The protective effect of O and increased risk associated with A blood types that they reported was subsequently investigated by many studies that returned varied results (e.g., [@doi:10.1002/jmv.27444; @doi:10.3389/fcimb.2020.00404; @doi:10.1182/bloodadvances.2020002623; @doi:10.1001/jamanetworkopen.2021.7429]; see [@doi:10.1007/s00277-021-04489-w] for a literature review).
+Support for higher and lower risk, respectively, of SARS-CoV-2 infection with A and O blood types was supported by a meta-analysis [@doi:10.1016/j.meegid.2020.104485].
+While the support for the association was independent of a mechanism, a possible relationship between ACE activity and blood type has been proposed [@doi:10.1177/2047487320922370] as has an effect on carbohydrate-carbohydrate interactions relevant to ACE2 binding [@doi:10.1016/j.mehy.2020.110155].
+This is the only candidate gene described that has received additional support from GWAS, as is discussed below.
+
+The COVID-19 literature related to candidate gene investigations demonstrates relatively low inter-study consistency in findings.
+In particular, sample size is a major challenge in designing these studies.
+However, for many traits, the relationships between genes and phenotypes are complex, and selecting which variants to sequence is not always straightforward
+.
+As a result, in the age of next-generation sequencing, discovery-driven studies have emerged as an alternative approach.
+
+##### Genome-Wide Association Studies
+
+Genome-wide association studies (GWAS) offers a discovery-driven approach that provides a different perspective than candidate gene studies.
+Instead of selecting a gene or variant _a priori_, in GWAS, a large number SNPs (usually several million) are evaluated at once to identify those most likely to vary in correlation with a trait of interest.
+Because of the large number of statistical tests, statistical power and multiple hypothesis testing are both very important considerations in executing GWAS, which have also struggled with issues related to replicability [@doi:10.1016/j.tig.2018.03.005].
+In cases such as COVID-19 where outcomes can differ among ancestry groups (likely for non-genetic reasons, as reviewed in [@individual-inequality]), it is especially important that GWAS samples be selected with attention paid to ancestry, as spurious associations can otherwise be identified with neutral markers indicative of ancestry itself [@doi:10.1038/nrg2813].
```

Comment:
```suggestion
In cases such as COVID-19 where outcomes can differ among ancestry groups (likely for non-genetic reasons, as reviewed in [@individual-inequality]), it is especially important that GWAS samples be selected with attention paid to ancestry, as incorrect or misleading associations can otherwise be identified with neutral markers indicative of ancestry itself [@doi:10.1038/nrg2813].
```